src/components/ProductCard/ProductCard.jsx

```diff
@@ -28,14 +34,21 @@ const ProductCard = ({ item }) => {
 
       {cartState.cartList.find((cartItem) => cartItem._id === item._id) ? (
         <button className="card-add-btn">
-          <Link to="/AddToCart" className="link">
+          <Link
+            to="/AddToCart"
+            className="link"
+            onClick={(e) => e.stopPropagation()}
```

Comment:
Oh, this stopPropagation is new for me!

Reply:
For me too, just found that today in someone else code it's so useful.